engine/src/main/java/com/google/android/fhir/db/impl/dao/ResourceDao.kt

```diff
@@ -152,7 +172,18 @@ internal abstract class ResourceDao {
 
   @RawQuery abstract suspend fun countResources(query: SupportSQLiteQuery): Long
 
-  private suspend fun insertResource(resource: Resource): String {
+  suspend fun insertResourceLocal(resource: Resource, timeOfChange: Instant) =
+    insertResource(resource, timeOfChange)
+
+  // Since the insert removes any old indexes and lastUpdatedLocal (data not contained in resource
+  // itself), we extract the lastUpdatedLocal if any and then set it back again.
+  private suspend fun insertResourceRemote(resource: Resource) =
```

Comment:
can we update this to:
```suggestion
  private suspend fun insertRemoteResource(resource: Resource) =
```

Reply:
Done